### PR TITLE
[VTCCA-3344] - fixed file upload error message issue

### PR DESCRIPTION
**Pull request by jon-pike**

### CODE REVIEW

**Comment by PeterPerhac** [comment]
👍 LGTPP

File: app/assets/javascripts/src/fileUpload.js [hunk 1]
```diff
@@ -19,10 +19,13 @@
         $element.change(function(){
             $('#error-summary').remove();
             $('#uploadFile').attr('disabled','disabled');
+            var file = this.files[0];
 
-            $(this).after('<div class="message-warning" id="message-warning"><p>Please wait whilst your file is uploading. This may take some time.</p></div>');
 
-            var file = this.files[0];
+            if(file.type != "" && file.type != undefined){
```
Comment (darrell-plant) [inline app/assets/javascripts/src/fileUpload.js:25]: you could try this: https://stackoverflow.com/questions/5515310/is-there-a-standard-function-to-check-for-null-undefined-or-blank-variables-in

**Comment by darrell-plant** [comment]
otherwise LGTM 👍

File: app/assets/javascripts/src/fileUpload.js (hunk 1 above)
Comment (jon-pike) [reply app/assets/javascripts/src/fileUpload.js]: Thanks Darrell - works a treat

**Pull request merged**